pkg/api/datasources_test.go

```diff
@@ -225,15 +225,16 @@ func TestUpdateDataSource_InvalidJSONData(t *testing.T) {
 }
 
 // Using a team HTTP header whose name matches the name specified for auth proxy header should fail
-func TestUpdateDataSourceTeamHTTPHeaders_InvalidJSONData(t *testing.T) {
+func TestAddDataSourceTeamHTTPHeaders(t *testing.T) {
 	tenantID := "1234"
 	testcases := []struct {
```

Comment:
Given we only want to test that we cannot update the rules anymore, I'd drop the table test structure.